vkdgen.py

```diff
@@ -113,7 +113,9 @@ def convertTypeArray(typ, name):
 class DGenerator(OutputGenerator):
 	def __init__(self, errFile=sys.stderr, warnFile=sys.stderr, diagFile=sys.stderr):
 		super().__init__(errFile, warnFile, diagFile)
-	
+		self.enumConstants = []
+		self.funcNames = []
+

```

Comment:
> Why should a D user have to when the same information is available in D modules?
Because Vulkan is a C API.

I don't think that prettifying the machine-generated bindings is a high priority when I could be working on cleaning up the code and adding platform support (especially since the C headers are 99% the same; there is little gain). If you think so, then I'll take a pull request.